broker-daemon/block-order-worker/index.js

```diff
@@ -1,17 +1,25 @@
 const EventEmitter = require('events')
 const { promisify } = require('util')
 const safeid = require('generate-safe-id')
+
 const { BlockOrder, Order, Fill } = require('../models')
 const { OrderStateMachine, FillStateMachine } = require('../state-machines')
+const {
+  Big,
+  getRecords,
+  SublevelIndex,
+  events: { BlockOrderWorkerEvents }
```

Comment:
I don't think this should be in utils since it's specific to this module

Reply:
it is shared between modules

Reply:
@treygriffith we are removing this in the next iteration of the PR